### DISCUSSION TITLE
Section 3: solution to Latex Errors on Linux platforms

### DISCUSSION THREAD

**Discussion by kwamentw** [Cyfrin/security-and-auditing-full-course-s23#14] [Ideas]
To install laTex follow Patrick's link in the repo to install after installing you export your path like this:

```bash
export PATH="/usr/local/texlive/2023/bin/x86_64-linux"
```
after installing and exporting your path, when you ran Patrick's command to generate the pdfs you might ran into errors like

```bash
LaTeX Error: File `sourcecodepro.sty' not found
Package fontenc Error: Encoding file `ly1enc.def' not found
```
I realised these missing packages are fonts i might be wrong 

To get rid of the error you have to find the package in which the missing file is found in to install
This is how i got rid of mine there are other ways too

```bash
tlmgr install sourcecodepro
tlmgr install ly1
```

**Comment by cromewar** [comment]
Thanks for this @kwamentw

**Reply by kwamentw** [reply]
You're welcome! Upvote so it can help others.

**Reply by PatrickAlphaC** [reply]
Awesome!!

**Comment by denissosnowsky** [comment]
In case `tlmgr` command does not work for you because of the old version error (some OS have installed older version of texlive by default), make sure you added `/usr/local/texlive/2023/bin/x86_64-linux` at the beginning of the `PATH` variable:
Correct:
```
export PATH="/usr/local/texlive/2023/bin/x86_64-linux:$PATH"
```
Not correct:
```
export PATH="$PATH:/usr/local/texlive/2023/bin/x86_64-linux"
```
